pkg/test/resourcefixture/testdata/basic/accesscontextmanager/v1beta1/accesscontextmanagerserviceperimeterresource/create.yaml

```diff
@@ -12,11 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: accesscontextmanager.cnrm.cloud.google.com/v1alpha1
+apiVersion: accesscontextmanager.cnrm.cloud.google.com/v1beta1
 kind: AccessContextManagerServicePerimeterResource
 metadata:
   name: accesscontextmanagerserviceperimeterresource-${uniqueId}
 spec:
   perimeterNameRef:
-    name: accesscontextmanagerserviceperimeter-${uniqueId}
-  resource: projects/${projectNumber}
+    # Referencing a resource name leads to recursive reference and KCC does not support the feature for now.
+    # Please use external reference instead.
+    external: "accessPolicies/578359180191/servicePerimeters/restrict_all_test"
```

Comment:
Is `578359180191` a pre-created test project that we will unlikely touch? Do we have a recurring job to ensure this dependent resource is ready?

It looks like the same project is used by other test cases - will there be any racing conditions when those test cases are running in parallel?

Reply:
`578359180191` is a pre-created access policy under our test organization. In the API doc, only one policy per org is allowed([ref](https://cloud.google.com/access-context-manager/docs/create-access-policy)). I think it's expected that all other resources in this org will use this same policy, so I won't worry too much about the recurring job.
Let me know if you have other opinions!

Reply:
Gotcha. I think we should keep an eye on the test cases that dependent on this AccessContextManagerAccessPolicy in the periodic jobs after the change is merged.